README.md

```diff
@@ -132,3 +132,45 @@ First edit the `docker-compose.yml` file with any configuration keys (as describ
 cd examples
 docker compose up -d
 ```
+
+### Podman Compose
+
+Podman Compose works almost identically to Docker's version but watchtower is not the recommended auto-update method.  To achive a similar effect:
+#### One Time Setup
+1. (as root) Set up the base systemd unit file for containers:
+`sudo /usr/local/bin/podman-compose systemd --action create-unit`
+1. (as your user) Configure the auto-update timer:
+`systemctl --user enable --now podman-auto-update.timer`
+1. (as your user) Create your directory to hold compose files:
+`mkdir -p ~/containers`
+
+#### For each container:
+1. Set up a directory to hold the container and other files:  
```

Comment:
It's been a while since I've done podman, I admit. But why do they need to do this for "each container"?
If I recall correctly, podman-compose supports the scale parameter.